.papr.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -xeuo pipefail
+
+export GOPATH=$HOME/gopath
+export PATH=$HOME/gopath/bin:$PATH
+export GOSRC=$HOME/gopath/src/github.com/projectatomic/libpod
+
+(mkdir -p $GOSRC && cd /code && cp -r . $GOSRC)
+
+dnf install -y \
```

Comment:
See also https://github.com/ostreedev/ostree/blob/master/ci/libbuild.sh - once something is packaged then you can just do `pkg_install_builddeps libpod`.